dobot_arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
  gains:
    joint_1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_3:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_4:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
dobot_hand_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint_grip
  gains:
    joint_grip:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
